### PR TITLE
[IMP] Remove utf-8 headers on Odoo >= 11.0

### PR DESCRIPTION
**Pull request by Yajo**
Obeying @moylop260 at https://odoo-community.org/groups/contributors-15/contributors-64878 😄 

@Tecnativa

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -560,8 +560,9 @@ except ImportError:
 
 ### Idioms
 
-* Each python file should have
-  ``# coding: utf-8`` or ``# -*- coding: utf-8 -*-`` as first line
+* Each python 2 (Odoo < 11.0) file should have
+  ``# coding: utf-8`` or ``# -*- coding: utf-8 -*-`` as first line. Python 3
```
Comment (elicoidal) [inline CONTRIBUTING.md]: Suggestion:
```
* For Python 2 (Odoo < 11.0), all python files should contain ``# coding: utf-8`` or ``# -*- coding: utf-8 -*-`` as first line. 
* For Python 3 (Odoo >= 11.0), no need for utf-8 coding line as this is implicit.
```

**Comment by NL66278** [comment]
Question: I understand Odoo 11 will be compatible with both python 2.7 and 3.x. Is it not better to wait with removing the utf header until we have an odoo version requiring python >= 3.x?

**Comment by leemannd** [comment]
@NL66278 Oddo11 remains actually compatible with python 2.7. But they won't officially support it and they officially released it with requirements of python >=3.x .

**Comment by Yajo** [comment]
@NL66278 Check this out: https://github.com/odoo/odoo/commit/961b7820deabaddfd01fbb7d380feb854f354b6d#commitcomment-24324033 (it was the initial proposal, it's now a fact, confirmed at the Odoo Experience)

**Comment by moylop260** [comment]
Thanks
I’ll work on pylint-odoo to disable it this check for 11.0 modules

**Comment by Yajo** [comment]
Or invert the check, instead of disabling.

**Pull request merged**